modules/80_optimization/nash/declarations.gms

```diff
@@ -92,11 +92,20 @@ p80_count                                  "count regions with feasible solution
 p80_eoWeights_fix(all_regi)                "default and fallback weighting factors"
 
 p80_SolNonOpt(all_regi)                    "solve status"
-p80_taxrev_dev(ttot,all_regi)              "deviation of tax revenue normalized by GDP if taxes did not converge"
 
 pm_fuExtrForeign(ttot,all_regi,all_enty,rlf) "foreign fuel extraction"
-p80_taxrev_dev(ttot,all_regi)                "deviation of tax revenue in percent GDP"
-;
+
+p80_convNashTaxrev_iter(iteration,ttot,all_regi) "deviation of tax revenue in percent GDP per iteration"
```

Comment:
as some other convergence deviation parameters include the `_dev` suffix, maybe we should adopt this on the naming convention for these convergence deviation parameters also.

Reply:
I think, here it is more important that it it a Nash con. criterion, so that it includes `convNash`

Reply:
I didn't ask for changing the `convNash` part of the parameter name. My request was to add the _dev suffix (e.g. p80_convNashTaxrev_dev_iter) like it is used in other existent convergence check parameters (e.g. `p80_emiMktTarget_dev_iter`, `p80_implicitQttyTarget_dev_iter`, `p80_globalBudget_dev_iter`).